northstar/src/runtime/container_key.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2021 ESRLabs
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+use super::{Name, RepositoryId, Version};
+use serde::{Deserialize, Serialize};
+use std::{
+    convert::TryFrom,
+    fmt::{self, Display},
+    sync::Arc,
+};
+
+#[derive(Clone, Eq, PartialEq, Debug, Hash, Serialize, Deserialize)]
+pub struct ContainerKey {
```

Comment:
would not call this `Key`. yes it's used as a key in a hashmap but conceptually it is still a container id

Reply:
The name key has been chosen because it's also used as the key for maps. Agree that `ContainerKey` is long and not ideal. My personal perference is `Container` and the current `Container` is renamed to `ContainerInstance`.

Thoughts?

Reply:
not sure...what it it is a unique identifier for a container-instance, yes...
but then *-Instance is some what redundant since a lot of things are instances of some kind.
Just leave it as `ContainerKey` or `ContainerId` then

Reply:
So `ContainerId` and `Container`? I'd like to get rid of `key`.

Reply:
Done in `8835f8a`